specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -200,7 +200,8 @@
         "weekly",
         "daily",
         "hourly",
-        "minutely"
+        "minutely",
+        "secondly"
```

Comment:
This isn't pertinent to the swagger review, but just feedback for your service.

The term "minutely" is not common--I've never heard it used an had to look it up to verify it's actually a word in the dictionary (surprised to find that it is). However, the term "secondly" does NOT mean what is implied here (i.e. "per second"). It means "In second place". I would recommend deprecating these enum values with replacing them with either pattern:
`week, day, hour, minute, second` or `perWeek`, `perDay`, `perHour`, `perMinute`, `perSecond`. I would recommend the first. It's simpler and consistent with what I've seen in other services like Azure Monitor.